ui/prometheus-plugin/src/plugins/prometheus-time-series-query/PrometheusTimeSeriesQueryEditor.tsx

```diff
@@ -66,46 +59,41 @@ export function PrometheusTimeSeriesQueryEditor(props: PrometheusTimeSeriesQuery
     throw new Error('Got unexpected non-Prometheus datasource selector');
   };
 
-  return (
-    <Stack spacing={2}>
-      <FormControl margin="dense" fullWidth={false}>
-        {/* TODO: How do we ensure unique ID values if there are multiple of these? Can we use React 18 useId and
-            maintain 17 compatibility somehow with a polyfill/shim? */}
-        <InputLabel id="prom-datasource-label">Prometheus Datasource</InputLabel>
-        <DatasourceSelect
-          datasourcePluginKind={PROM_DATASOURCE_KIND}
-          value={selectedDatasource}
-          onChange={handleDatasourceChange}
-          labelId="prom-datasource-label"
-          label="Prometheus Datasource"
-        />
-      </FormControl>
-      <PromQLEditor
-        completeConfig={{ remote: { url: promURL } }}
-        value={query}
-        onChange={handleQueryChange}
-        onBlur={handleQueryBlur}
+  if (isExplore) {
```

Comment:
Does this isExplore condition need to be in prometheus-plugin this way? Can ui/explore import relevant lower components and rebuild the parts it needs, I’m thinking useState for managing active queries and interact with the query string directly